packages/gatsby-remark-images/src/index.js

```diff
@@ -219,6 +219,7 @@ module.exports = (
           // override options if it's an object, otherwise just pass through defaults
           options.withWebp === true ? {} : options.withWebp,
           pluginOptions,
+          { pathPrefix },
```

Comment:
Oh nice catch, I think we should use options instead of pluginOptions here. What do you think?

```
        args: _.defaults(
          { toFormat: `WEBP` },
          // override options if it's an object, otherwise just pass through defaults
          options.withWebp === true ? {} : options.withWebp,
          options,
        ),
```

Reply:
Yes, I think it is an even better solution because any change or extension to 'options' will also apply to the creation of WebP images.

Reply:
I just changed the original commit for another with your proposal.